src/commands.rs

```diff
@@ -3376,11 +3376,15 @@ fn cmd_op_log(
             })?;
             formatter.write_str(" ")?;
             formatter.with_label("time", |formatter| {
-                formatter.write_str(&format!(
-                    "{} - {}",
-                    self.format_timestamp(&metadata.start_time),
-                    self.format_timestamp(&metadata.end_time)
-                ))
+                let start = self.format_timestamp(&metadata.start_time);
+                let end = self.format_timestamp(&metadata.end_time);
+                formatter.write_str(
+                    &(if start != end {
+                        format!("{start} - {end}",)
+                    } else {
+                        start
+                    }),
+                )
```

Comment:
Maybe we should only do this only when using relative timestamps? I guess it doesn't really matter in practice because the absolute timestamps are practically never going to be to be the same (very few transactions finish within a millisecond). But since that also means that it's unnecessary to compare them in that case, maybe it's better to skip it? That would also make most tests unaffected.

Reply:
Sounds good.